src_modules/voice_commands/phrases.ts

```diff
@@ -1,17 +1,17 @@
-import { SimplePhrase } from "extcord";
+import { MessagePhrase } from "extcord";
 
-export const voiceCommandsEnabledPhrase = new SimplePhrase({
+export const voiceCommandsEnabledPhrase = new MessagePhrase({
```

Comment:
This is not a correct way to use MessagePhrase. See in the example phrases.ts file on how to use them.

Reply:
This is still incorrect. Here is an example on how to create MessagePhrase:

export const voiceJoinPhrase = new MessagePhrase(
    {
        description: "Shown when the voice channel is joined",
        name: "voiceJoin",
    },
    "Joined the voice channel.",
    {
        timestamp: false,
        title: "Joined the voice channel."
    },
    {},
);